code/modules/mob/living/carbon/monkey/monkey.dm

```diff
@@ -158,6 +158,9 @@
 	return 1
 
 /mob/living/carbon/monkey/can_use_guns(obj/item/G)
+	if(G.trigger_guard == TRIGGER_GUARD_NONE)

```

Comment:
Since this is a living helper common ones should be there.

Reply:
living has G.trigger_guard != TRIGGER_GUARD_ALL so there is no need i guess